modules/ragstack/pages/quickstart.adoc

```diff
@@ -0,0 +1,479 @@
+= Quickstart
+
+The {ragstack} project combines the intelligence of large language models with the agility of stream processing to create powerful Generative AI applications.
+
+This guide will help you build and deploy a {ragstack} application to Astra Streaming.
+
+== Install {ragstack} CLI
+
+Install the {ragstack} CLI:
+[source,bash]
+----
+brew tap ragstack/ragstack
+brew install ragstack
+----
+
+For more on the {ragstack} CLI, see https://docs.langstream.ai/installation/langstream-cli[{ragstack} CLI^]{external-link-icon}.
+
+== Enable {ragstack} in Astra
+
```

Comment:
Can we add a note about this being in private preview and that you should contact DataStax sales to get it enabled.

Reply:
Added